tdm-fe/src/app/components/layout/WebServicesDescription.tsx

```diff
@@ -0,0 +1,31 @@
+import '~/app/components/layout/scss/WebServicesDescription.scss';
+import Container from '@mui/material/Container';
+
+const WebServicesDescription = () => {
+    return (
+        <div id="ws-description">
+            <Container className="text">
+                <p>Par exemple :</p>
+                <ul>
+                    <li>
+                        chargez vos données au format CSV : VosData.csv (dans le cas de corpus ISTEX on pourra utiliser
```

Comment:
Une espace fine insécable..
```suggestion
                        chargez vos données au format CSV&#8239;: VosData.csv (dans le cas de corpus ISTEX on pourra utiliser
```